### PR TITLE
implement api/screenshot route

### PR DESCRIPTION
**Pull request by ruben1**

### CODE REVIEW

File: server/api/screenshot/index.js [hunk 1]
```diff
@@ -17,7 +23,7 @@ module.exports = function(router) {
    * /api/user/:username/screenshot/:id
    */
   router.route('/api/user/:username/screenshot/:id')
-    // get the screenshot
+    // get an specific screenshot from a user
```
Comment (AndrewSouthpaw) [inline server/api/screenshot/index.js]: grammar: get **a** specific

File: server/api/screenshot/index.js [hunk 2]
```diff
@@ -3,10 +3,16 @@ var auth = require('../../config/auth');
 
 module.exports = function(router) {
   /**
+   * /api/screenshot
+   */
+  router.route('/api/screenshot')
+    //get all screenshots
+    .get(screenshotController.retrieveAll);
+  /**
```
Comment (AndrewSouthpaw) [inline server/api/screenshot/index.js:13]: Separate out with a carriage return for clarity.

File: server/api/screenshot/screenshotController.js [hunk 3]
```diff
@@ -2,6 +2,13 @@ var Screenshot = require('./screenshotModel');
 var User = require('../user/userModel');
 var takeScreenshot = require('../../screenshotCapture/script.js');
 
+exports.retrieveAll = function(req, res, next){
```
Comment (AndrewSouthpaw) [inline server/api/screenshot/screenshotController.js:5]: Not that it's really implemented, but I think we should filter by the privacy setting. As long as it defaults to true right now in the schema, adding the line won't negatively impact anything

Comment (ruben1) [inline server/api/screenshot/screenshotController.js:5]: sure

**Comment by AndrewSouthpaw** [comment]
LGTM.

**Pull request merged**